src/by/incubator/task2/JarTask.java

```diff
@@ -1,23 +1,23 @@
-package by.incubator.task2;
-
-import by.devInc.module2.task1.Main;
-import java.net.MalformedURLException;
-import java.net.URL;
-import java.net.URLClassLoader;
-
-public class JarTask {
-
-    public static void main(String[] args) throws MalformedURLException, ClassNotFoundException {
-        URL [] classLoaderUrls = {new URL("file:///H:\\IdeaProjects\\DevIncJavaTasks\\out\\example.jar")};
-        URLClassLoader urlClassLoader = new URLClassLoader(classLoaderUrls);
-        Class uploaded = urlClassLoader.loadClass("by.devInc.module2.task1.Main");
-        try {
-            Main instance = (Main) uploaded.newInstance();
-            instance.main(new String[]{});
-        } catch (InstantiationException e) {
-            throw new RuntimeException(e);
-        } catch (IllegalAccessException e) {
-            throw new RuntimeException(e);
-        }
-    }
-}
+//package by.incubator.task2;
```

Comment:
If this class doesn't need, it should be remove, not committed

Reply:
fixed